### ISSUE TITLE
Change the Request format to match RFC 2119

### ISSUE THREAD

**Issue by inverted-capital** [dreamcatcher-tech/dreamcatcher-tech.github.io#41]
[RFC 2119](https://tools.ietf.org/html/rfc2119) describes some of the RFC process, and in particular describes the key words we use in our Requests in terms of MUST, MUST NOT, etc.

After looking at [some other examples](https://semver.org/#semantic-versioning-specification-semver) of usage of the RFC format around the internet, it becomes clear that these words are all blended together.

Should we perhaps adopt this format, rather than our current listing of MUST, MUST NOT, etc all separately, for the purpose of being familiar to others on the internet, rather than our own custom format ?

**Comment by inverted-capital** [comment]
This describes the actual process: https://www.rfc-editor.org/rfc/rfc2026.txt  Apologies I've not yet read it

**Comment by TemperBead** [comment]
Fascinating, and yes, very much so.  

I see also from it's usage that these terms are used in each clause, at least one per clause.  That's a better format that the one I came up with with the separate sections, because it allows grouping of clauses around concepts and not around permissions.

**Comment by TemperBead** [comment]
Suggest we hold off on an R10 rewrite.  Reasoning below.

On grouping, I do prefer grouping around concepts.  In some ways I've tried to do that in [R00 Request Template](https://github.com/dreamcatcher-tech/dreamcatcher-tech.github.io/blob/master/website/nfas/Requests/R00.md) but perhaps in a clunky way.  The top level grouping is within Detailed Description, Tactical Intents 1, 2... which is the concept level, then the strength of requirement.  The main difference between this and [RFC 2119](https://tools.ietf.org/html/rfc2119) is that the strength of requirement is in-line rather than by sub-section.  I like that.  Can't immediately say why, but it seems to simplify the format but keep the info.

Now, the interesting thing is that you have a preference for strength of requirement first.  I don't see any honest way other than the market to decide which one is relevant, or whether both are relevant in different circumstances, or whether both are relevant in all circumstances but where one or the other is easier for the author to get their head round.  I really don't think we should dictate one format to rule them all, and pretty sure you'll agree, but that leads to two questions: inheritance and usage.

On inheritance, this is a great opportunity to test that concept.  

I'd suggest we test that on the templates.  [R00 Request Template](https://github.com/dreamcatcher-tech/dreamcatcher-tech.github.io/blob/master/website/nfas/Requests/R00.md) is what we have at the moment.  Can we break that down to a 'minimal viable template', then build up at least one level of inheritance where that MVT is used as base, and one leaf from there goes to Concept first, the other strength of requirement first?  Perhaps theres a third, QDiRT - Quick and Dirty Request Template?  That still is well formatted, but has the least amount of overhead in getting QA to agree that it can go into a pool.

On usage, my gut says that we don't impose a rule that there's only one template (see above) or given that there's more than one format, that a pool can only accept one format.

I'd also propose that it comes down to the pool to decide and advertise what formats they will accept.  Don't see any reason why that can't be multiple.  For example, a pool could advertise two formats - simple and quick, but nevertheless well formed, and a fuller format.  

Propose we test this idea, and the concept of inheritance, generating templates.  Currently we've got [R00 Request Template](https://github.com/dreamcatcher-tech/dreamcatcher-tech.github.io/blob/master/website/nfas/Requests/R00.md)  and [I00 Idea Template](https://github.com/dreamcatcher-tech/dreamcatcher-tech.github.io/blob/master/website/nfas/Ideas/I00.md) inside nfas, but that's probably not the right place for them?  

Now, the discipline of this is, we need to generate a Request and Idea for multiple formats, and we haven't completed the R10/I11 cycle.  Think we should get those over the line (as in, Outputs produced and accepted), then come back to this issue and generate a new Request/Idea for this.  It won't impact on the R10 process as that devolves the template structure issue to QA, so we shouldn't need to redo the output.  NB also means that a rewrite of R10 is unnecessary.

**Comment by inverted-capital** [comment]
Grouping by heading is definitely a subset of grouping around concept.

Yes lets just stick to our original RFCMSM format for now, and implement a means for any format to be specified using inheritance later.

**Comment by TemperBead** [comment]
Agreed.  You raised this so will leave it to you to close if we're good.  Good this side.

**Comment by inverted-capital** [comment]
I'll close this once we have a Request in the system to do this task.
